pywb/apps/rewriterapp.py

```diff
@@ -366,7 +376,12 @@ def render_content(self, wb_url, kwargs, environ):
                                                    config=self.config))
 
         cookie_rewriter = None
-        if self.cookie_tracker:
+        if self.cookie_tracker and cookie_key:
```

Comment:
Running code inspection on this file reports that cookie_key maybe referenced before assignment here

I would feel better if we assign `cookie_key = None` after line 275 as is done with the variable `setcookie_header`